privacyidea/lib/tokens/vasco.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+#
+# 2018-01-15 friedrich.weber@netknights.it
+#            Remove unused routines, redesign serialization step,
+#            adapt to privacyIDEA, refactoring
+#
+#    LinOTP - the open source solution for two factor authentication
+#    Copyright (C) 2010 - 2017 KeyIdentity GmbH
+#
+#    This program is free software: you can redistribute it and/or
+#    modify it under the terms of the GNU Affero General Public
+#    License, version 3, as published by the Free Software Foundation.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the
+#               GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#    E-mail: linotp@keyidentity.com
+#    Contact: www.linotp.org
+#    Support: www.keyidentity.com
+""" VASCO library binding """
+
+import logging
+
+# from ctypes import *
+from ctypes import CDLL, create_string_buffer
+from ctypes import Structure
+
+from ctypes import byref
+from ctypes import c_ulong
+from ctypes import c_char
+from ctypes import c_byte
+
+from flask import current_app
+
+from privacyidea.lib.error import ParameterError
+
+__all__ = ["vasco_otp_check"]
+
+
+log = logging.getLogger(__name__)
+
+vasco_dll = None
+
+try:
+    vasco_library_path = current_app.config.get("VASCO_LIBRARY")
```

Comment:
Can we call this PI_VASCO_LIBRARY?

As pi.cfg also contains standard parameters like SECRET_KEY or SQLALCHEY_URI, I tried to prefix our own parameters with PI_ (unfortunately not consistant)